examples/04_example_patchextraction.ipynb

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
<p>Please run the notebook with tiatoolbox 0.7.0. One way is to run on Google Colab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Tested </p>